### PR TITLE
[BE20-246] Fixed tests and docker image config

### PR DESCRIPTION
**Pull request by k-vinogradov**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,38 +1,42 @@
 FROM ubuntu:xenial
-ENV PYTHON_VERSIONS='python2.7 python3.4 python3.5 python3.6 python3.7' \
+ENV PYTHON_VERSIONS='python3.7' \
```
Comment (DrLoboto) [inline Dockerfile:2]: The original idea was in returning our work in master SendGrid repo. Why all supported versions Python are replaced by the single one?

File: tests/test_unit.py [hunk 2]
```diff
@@ -164,25 +164,25 @@ def test__getattr__(self):
         # Test GET
         mock_client._url_path += ['test']
         r = mock_client.get()
-        self.assertEqual(r.status_code, 200)
+        self.assertEqual(r.response_code, 200)
```
Comment (DrLoboto) [inline tests/test_unit.py:167]: Why different attribute name? Return types must NOT be changed.

Comment (k-vinogradov) [reply tests/test_unit.py:167]: It's the only right attribute name. The problem is that the original tests don't work at all. Seems `python-http-client` repo by SendGrid isn't maintained so good.

File: Dockerfile (hunk 1 above)
Comment (k-vinogradov) [reply Dockerfile:2]: SendGrid guys test only python2.7. I've just replaced it with python3.7. Should I extend the tests for the BOTH 2.7 and 3.x Python version?

**Pull request closed without merging**